etcdmgmt/etcd-mgmt_test.go

```diff
@@ -20,10 +48,11 @@ func TestStartETCD(t *testing.T) {
 	etcdCtx, err := StartETCD()
 	tests.Assert(t, err == nil)
 	etcdCtx.Kill()
+	etcdCtx.Wait()
 }
 
 func TestWriteETCDPidFile(t *testing.T) {
-	cmd := exec.Command("etcd")
+	cmd := formETCDCommand()
```

Comment:
In formETCDCommand there is no code where an error would be returned, only one possibility is where os.Hostname() would fail but since we are calling log.Fatal the whole test would abort.